ktor-server/ktor-server-host-common/jvm/test/io/ktor/tests/hosts/ReceiveBlockingPrimitiveTest.kt

```diff
@@ -86,6 +87,10 @@ class ReceiveBlockingPrimitiveTest {
         override val response: BaseApplicationResponse
             get() = error("Shouldn't be invoked")
 
+        override fun onCallFinish(handler: (Throwable?) -> Unit) {
+            throw IllegalAccessException("onCallFinish is not available for TestCall")
```

Comment:
`throw IllegalAccessException` -> `error()`

btw why it's not available?

Reply:
It's not available because there is no `job` available here
And it doesn't make much sense to do this in this `TestCall` class as it is used only in one single place in a test, and it's private anyway